code/modules/mob/living/carbon/xenomorph/Castes/Ravager.dm

```diff
@@ -48,6 +49,13 @@
 		/datum/action/xeno_action/second_wind,
 		)
 
+/mob/living/carbon/Xenomorph/Ravager/Stat()
+	. = ..()
+	if(!.)
+		return FALSE
+
+	stat(null, "Rage: [rage] / [RAVAGER_MAX_RAGE]")
+	return TRUE
```

Comment:
Is this needed? We are already preserving the return value in `.`
Same regarding the earlier `return FALSE`. If the `if` statement isn't true, then it will return null or FALSE anyway.

Reply:
Uhh, I see I dun goofed by asking both preserving the return and to return TRUE or FALSE. My bad here.